packages/ckeditor5-build-classic/package.json

```diff
@@ -26,6 +26,7 @@
   ],
   "devDependencies": {
     "@ckeditor/ckeditor5-adapter-ckfinder": "^20.0.0",
+    "@ckeditor/ckeditor5-alignment": "^20.0.0",
```

Comment:
I'm assuming you're leaving the caret `^` on purpose in this case... so that we get the latest every time we build?